powerhome/playbook
# [PLAY-762] Converting Timeline to Typescript

**What does this PR do?**
Converting Timeline kit to Typescript.

**How to test?**
1. Go to Timeline kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
